### DISCUSSION TITLE
Text changes border

### DISCUSSION THREAD

**Discussion by sheepdoggie** [freeplane/freeplane#540] [Help] [answered]
**Describe the bug**
Entering a specific sequence of characters into the text of a node causes the node border to appear in a specific color.

**To Reproduce**
1. Enter schlimm

**Expected behavior**
The border should not be effected by the text.

**Mind map files**
[Freeplane Bug Report -- 08-05-2022.mm.zip](https://github.com/freeplane/freeplane/files/9269201/Freeplane.Bug.Report.--.08-05-2022.mm.zip)

**Screenshots**
![Screenshot_2022-08-05_06-45-04](https://user-images.githubusercontent.com/70542235/183091935-865b5dc6-e93c-4efb-8ced-cb82d9b45015.png)

**Desktop (please complete the following information):**
 - OS:Linux Mint 5.15.0-43-generic #46~20.04.1-Ubuntu SMP Thu Jul 14 15:20:17 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
 - Version Freeplane 1.10.3
 - Java Version: 11.0.14

**Additional context**

**Comment by dpolivaev** [comment, marked as answer]
It's not a bug. You probably used the filter toolbar to activate the option `Filter->Highlight all matching nodes`.
